store/src/java/com/zimbra/cs/util/ProxyConfGen.java

```diff
@@ -1106,6 +1107,107 @@ public void update() throws ServiceException {
     }
 }
 
+class OnlyOfficeDocServiceServersVar extends ProxyConfVar {
+
+    private final String defaultHost = "zimbra.com";
+
+    public OnlyOfficeDocServiceServersVar() {
+
+        super("web.upstream.onlyoffice.docservice", null, null, ProxyConfValueType.STRING, ProxyConfOverride.CUSTOM,
+                "List of upstream HTTPS servers towards docservice port used by Web Proxy ");
+    }
+
+    @Override
+    public void update() throws ServiceException {
+        StringBuffer sb = new StringBuffer();
+        // check if zimbraDocumentEditingHost has been set
+        // if it is set, it means onlyoffice is to be deployed on a single
```

Comment:
comment here is this a assumptions if zimbraDocumentEditingHost is set there is single onlyoffice host?
zimbraDocumentEditingHost is server config so can be different for each server

Reply:
The config is as such : 
1. We set the onlyoffice server in `zmprov mcf zimbraDocumentEditingHost <host>` on the box where nginx is running.
2. For Onlyoffice on each mailbox : we do not set it.

Is there any other way of setting it other than `zmprov mcf zimbraDocumentEditingHost <host>`?
If we set it in this way, won't it be reflected on all servers.

Reply:
It was a global, which has been changed to server level now.